apps/site/lib/predicted_schedule.ex

```diff
@@ -84,6 +84,8 @@ defmodule PredictedSchedule do
     |> Enum.sort_by(sort_fn)
   end
 
+  defp create_map({:error, _error}), do: %{}
```

Comment:
Could you add logging here too? I've seen quite a few predicted schedules errors in the past and it could be useful.